ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -19,8 +19,9 @@ class BottomSheetDemoController: UIViewController {
         optionTableView.separatorStyle = .none
         view.addSubview(optionTableView)
 
-        let bottomSheetViewController = BottomSheetController(contentView: personaListView)
+        let bottomSheetViewController = BottomSheetController(headerContentView: headerView, expandedContentView: personaListView)
```

Comment:
this direction is bit concerning. I have seen variation of bottomsheet which only the gripper is shown behind the tabbarview and expand the sheet accordingly

Reply:
seems like this kind of initialization is fine for commandingcontroller which has clear division of headercontentview and expanded content view

Reply:
What if I made the header optional? That should cover that usage scenario.

Reply:
I think forcing this type of division through the bottom sheet API is good, because it lets us enforce proper handling of the bottom safe area. Anything that shows a persistent sheet has some notion of a header (whatever shows in the collapsed state). And in the scenario you described above, there's just no header, which still fits this pattern.